doc/source/fibo_speed.rst

```diff
@@ -207,12 +140,16 @@ Or, graphically:
 
```

Comment:
I will need to also update the table and graphs about the cached based function.